### ISSUE TITLE
luarocks support

### ISSUE THREAD

**Issue by mkozjak** [flaviogrossi/nginx-lua-upload-module#1]

**Comment by flaviogrossi** [comment]
This would be nice to have, but unfortunately i'm a little busy lately, so it may require some time.

Have you considered or tried adding support for this and opening a pull request?

**Comment by mkozjak** [comment]
@flaviogrossi Nope, no time, either. Decided to go without this library, also. Still, luarocks should be supported here. :)

**Comment by flaviogrossi** [comment]
I agree on that and i plan to do it.
Can i ask you what library are you using as a replacement?

**Comment by mkozjak** [comment]
This one: https://www.nginx.com/resources/wiki/modules/upload. Plus some custom "backend" lua code to actually handle the upload. :/
